### DISCUSSION TITLE
No data in consul

### DISCUSSION THREAD

**Discussion by gerardkr** [edgexfoundry/community#215] [Q&A]
I'm trying to run EdgeX natively.  Redis is up and so is consul.  But when I start core-metadata I get a bunch of warnings like those below.  This doesn't match the instructions and it seems like a problem in any case.  Is there a missing step in the instructions where I am supposed to load data into consul or redis? 

level=WARN ts=2023-11-27T02:29:09.647411Z app=core-metadata source=config.go:1056 msg="did not get boolean from config provider for edgex/v3/core-common-config-bootstrapper/IsCommonConfigReady: strconv.ParseBool: parsing \"\": invalid syntax"

**Comment by lenny-goodell** [comment]
@gerardkr What version are you running?
Have you read this guide?
https://docs.edgexfoundry.org/3.1/getting-started/native/Ch-BuildRunNative/

**Comment by lenny-goodell** [comment]
BTW, the error sounds like you have not run the core-common-config-bootstrapper service prior to running Core Metadata.

**Reply by gerardkr** [reply]
@lenny-intel you are exactly right.  I found the service in the docker-compose.yaml file. 

I'm looking at Napa documentation:  https://docs.edgexfoundry.org/3.1/getting-started/native/Ch-BuildRunOnLinuxDistro/ starting about 1/4 down the page with "Run EdgeX".  I suggest it should say  
1 - Start redis 
2 - Start consul 
3 - (Missing) run core-common-config-bootstrapper
4 - And then talks about firing up all the individual services.

But there is no mention of needing to run core-common-config-bootstrapper at step 3 above.  

Since this "Run EdgeX" seems to imply you can run everything NATIVELY, I suggest we add a note about core-common-config-bootstrapper.  

Thanks so much for your help.

**Reply by lenny-goodell** [reply]
@gerardkr , that service is new in EdgeX 3.x. Please create an issue on edgex-docs for this.
